app/services/context_service.rb

```diff
@@ -0,0 +1,31 @@
+class ContextService
+  attr_accessor :context
+
+  def initialize(context)
+    self.context = context
+  end
+
+  def with_context(&block)
+    switch_context(context, &block)
+  end
+
+  def as_org_admin(&block)
+    switch_context(org_admin_context, &block)
+  end
+
+  private
+
+  def org_admin_context
+    decoded = Base64.urlsafe_decode64(context["headers"]["x-rh-identity"])
+    decoded.sub!(/"is_org_admin":false/, '"is_org_admin":true')
```

Comment:
In case we may need to change other attributes such as names to `sysadmin`, may we pass this substitute string (`is_org_admin`) as default parameters?

Reply:
This is consider a hack which might be a temporary solution. We don't need to generalize it.